### PR TITLE
[MODORDSTOR-321] - Logging Improvement - Configuration

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose

https://issues.folio.org/browse/MODORDSTOR-321
## Approach

The property file and all logging commands have been adjusted.
## Learning

@folio-org/acquisitions

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by azizbekxm** [comment]
image

**Comment by azizbekxm** [comment]
image

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -13,71 +13,77 @@
...
 
 ## Learning
 <!-- OPTIONAL
-  Help out not only your reviewer, but also your fellow developer!
+  Help out not only your reviewer but also your fellow developer!
   Sometimes there are key pieces of information that you used to come up
   with your solution. Don't let all that hard work go to waste! A
   pull request is a *perfect opportunity to share the learning that
-  you did. Add links to blog posts, patterns, libraries or addons used
+  you did. Add links to blog posts, patterns, libraries, or add-ons used
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:50]: Why did you add a comma here?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -13,71 +13,77 @@
...
   to solve this problem.
 -->
-@folio-org/acquisitions
 
-## Pre-Merge Checklist
-Before merging this PR, please go through the following list and take appropriate actions.
+## Pre-Merge Checklist:
+Before merging this PR, please go through the following list and take appropriate action.
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:55]: "äction" should be in the plural form

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -13,71 +13,77 @@
...
 
 - Does this PR meet or exceed the expected quality standards?
   - [ ] Code coverage on new code is 80% or greater
-  - [ ] Duplications on new code is 3% or less
+  - [ ] Duplications on new code are 3% or less
   - [ ] There are no major code smells or security issues
 - Does this introduce breaking changes?
-  - [ ] Were any API paths or methods changed, added or removed?
+  - [ ] Were any API paths or methods changed, added, or removed?
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:62]: Also please remove comma before or

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:50]: Both of them are correct
image

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:55]: ok but Grammarly recommends it
Before merging this PR, please review(go through) the following list and take appropriate action(actions).
image

File: PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:62]: ok

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Abdulkhakimov) [reply PULL_REQUEST_TEMPLATE.md:55]: We'll take a few actions from this list so it is not always incorrect to rely on grammarly

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Abdulkhakimov) [reply PULL_REQUEST_TEMPLATE.md]: It is preferable to make few changes to the current template so let's leave as it was

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:55]: ok, no problem, I will fix it

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md]: Ok, I have fixed

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -13,71 +13,77 @@
...
 
-## Pre-Merge Checklist
-Before merging this PR, please go through the following list and take appropriate actions.
+## Pre-Merge Checklist:
+Before merging this PR, please go through the following list and take appropriate action.
 
 - Does this PR meet or exceed the expected quality standards?
   - [ ] Code coverage on new code is 80% or greater
-  - [ ] Duplications on new code is 3% or less
+  - [ ] Duplications on new code are 3% or less
```
[the remark this answers is not in the document]
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:59]: ok, I have fixed

**Review by Abdulkhakimov — APPROVED** [review]
